tests/test_commands.py

```diff
@@ -443,6 +444,109 @@ def test_update_unit_sto4_start_time_exists_mock_prompt_True(
     ]
 
 
+# update_unit_quota
+
+
+def test_update_unit_quota_no_such_unit(client, runner, capfd: LogCaptureFixture) -> None:
+    """Try to update a non existent unit -> Error."""
+    # Create command options
+    command_options: typing.List = [
+        "--unit-id",
+        "unitdoesntexist",
+        "--quota",
+        2,  # 2 GB,
+    ]
+
+    # Run command
+    result: click.testing.Result = runner.invoke(update_unit_quota, command_options)
+    assert result.exit_code == 1
+    assert not result.output
+
+    # Get logging
+    _, err = capfd.readouterr()
+
+    # Verify message
+    assert f"There is no unit with the public ID '{command_options[1]}'." in err
+
+
+def test_update_unit_quota_confirm_prompt_False(client, runner, capfd: LogCaptureFixture) -> None:
+    """Start time already recorded. Answer no to prompt about update anyway. No changes should be made."""
```

Comment:
I don't completely understand, especially the first sentence. 
Maybe something like 
```suggestion
    """Unit quota should not be changed when answer to prompt is False."""
```